### PR TITLE
corrections OS-X and Windows

### PR DESCRIPTION
**Pull request by christophsax**
this is now working locally

### CODE REVIEW

**Comment by christophsax** [comment]
Tested both on mac and windows. Seems to work fine!

File: configure [hunk 1]
```diff
@@ -74,11 +74,13 @@ if [ ${platform} == "linux" ]; then
     chmod 0775 x13ashtml
     echo "*** downloaded Linux binary"
     cd ${cwd}
-elif [ if [ ${platform} == "osx" ]; then
+elif [ ${platform} == "osx" ]; then
```
Comment (eddelbuettel) [inline configure:77]: Ooops. Cut and paste late at night...

File: configure [hunk 2]
```diff
@@ -74,11 +74,13 @@ if [ ${platform} == "linux" ]; then
...
     cd inst
-    rm bin
     download https://github.com/x13org/x13prebuilt/raw/master/osx/x13ashtml.tar.gz
-    tar xfz x13ashtml.tar.gz
+    tar -zxvf x13ashtml.tar.gz
+    cp -r x13ashtml/bin .
+    cp -r x13ashtml/lib .
```
Comment (eddelbuettel) [inline configure:82]: Maybe as one `cp -r x13ashtml/bin x13ashtml/lib .`  ?

**Comment by eddelbuettel** [comment]
We have a few things to -- mention authors, copyrights, source links, ... in the repos, but I think you can now start to have seasonal depend on this.  

Nice :)

File: configure (hunk 2 above)
Comment (christophsax) [inline configure:82]: sure, feel free to change whatever seems appropriate

Comment (eddelbuettel) [inline configure:82]: maybe next pass :)

**Pull request merged**